eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServerV2.java

```diff
@@ -109,9 +110,20 @@ public Resource getCoreInfo(@PathVariable("variantCoreString") String variantCor
     }
 
     private void checkParameters(String variantCoreString, String annotationVepVersion,
-                                 String annotationVepCacheVersion, String species, String assembly)  {
+                                 String annotationVepCacheVersion, String species, String assembly) {
         if (!variantCoreString.contains(":")) {
             throw new IllegalArgumentException("Please describe a variant as 'sequence:location:reference:alternate'");
+        } else {
+            String[] regionId = variantCoreString.split(":");
+            if(regionId.length!=4) {
+                throw new IllegalArgumentException("VariantCoreString requires 4 fields, " + regionId.length +
```

Comment:
I think a more clear message would be again `Please describe a variant as 'sequence:location:reference:alternate'`

Reply:
Fixed :)